src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -223,25 +209,29 @@ protected void purchase(final boolean purchaseForBid, int PUsToSpend, final IPur
         final String stepName = step.getName();
         ProLogger.info("Simulating phase: " + stepName);
         if (stepName.endsWith("NonCombatMove")) {
+          ProData.initialize(this, dataCopy, playerCopy);
           final Map<Territory, ProTerritory> factoryMoveMap =
-              nonCombatMoveAI.doNonCombatMove(null, null, moveDel, dataCopy, playerCopy, true);
+              nonCombatMoveAI.doNonCombatMove(null, null, moveDel, true);
           if (storedFactoryMoveMap == null) {
             storedFactoryMoveMap =
-                ProSimulateTurnUtils.transferMoveMap(factoryMoveMap, unitTerritoryMap, dataCopy, data, player);
+                ProSimulateTurnUtils.transferMoveMap(factoryMoveMap, ProData.unitTerritoryMap, dataCopy, data, player);
```

Comment:
tranferMoveMap could grab the static value "ProData.unitTerritoryMap", then you could drop an arg from this method. call. Without looking at transferMoveMap in detail, that could make sense.